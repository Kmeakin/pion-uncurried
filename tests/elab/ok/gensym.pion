let f: fn(A: Type, a: A) -> A = fn(_: ?_, x: ?_) => x;