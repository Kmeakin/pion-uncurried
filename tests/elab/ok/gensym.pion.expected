let f: fn(A: Type, a: A) -> A = fn(_: Type, x: _) => x;
